config/importmap.rb

```diff
@@ -16,5 +16,14 @@
 pin "@material/base/foundation", to: "https://ga.jspm.io/npm:@material/base@14.0.0/foundation.js"
 pin "@material/dom/focus-trap", to: "https://ga.jspm.io/npm:@material/dom@14.0.0/focus-trap.js"
 pin "@material/dom/ponyfill", to: "https://ga.jspm.io/npm:@material/dom@14.0.0/ponyfill.js"
-pin "tslib", to: "https://ga.jspm.io/npm:tslib@2.5.0/tslib.es6.js"
+pin "tslib", to: "https://ga.jspm.io/npm:tslib@2.5.3/tslib.es6.mjs"
```

Comment:
I guess it's because there's a new version of the package, but I don't know if it was really need to update it as part of the addition of `@materials/menu`. I wonder if there's a flag similar to bundler's `--conservative` in `importmaps` 🤔